package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@remoteoss/json-schema-form",
+  "version": "0.0.0",
+  "description": "Headless UI form powered by JSON Schemas",
+  "author": "Remote.com <engineering@remote.com> (https://remote.com/)",
+  "license": "MIT",
+  "homepage": "https://json-schema-form.vercel.app/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remoteoss/json-schema-form.git"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "files": [
+    "dist",
+    "README.md",
+    "CHANGELOG.md",
+    "LICENSE.md"
+  ],
+  "scripts": {
+    "build": "node scripts/build.js",
```

Comment:
I should also say for me, scripts working:

- Build works
- Test works
- Lint works
- Format works

I haven't tried `release` as i'm afraid ill mess something up lol

Reply:
Nice that you mentioned `release` because it slipped my commits. It will be on its own PR #3